.eslintrc.js

```diff
@@ -44,6 +44,11 @@ module.exports = {
     '@typescript-eslint/no-non-null-assertion': 'off',
     '@typescript-eslint/no-this-alias': 'off',
     'func-names': 'error',
+    'import/first': 'error',
+    'import/newline-after-import': 'error',
+    'import/no-duplicates': 'error',
+    'import/order': ['error', {'newlines-between': 'always'}],
```

Comment:
Please make sure this is done by some automatic tool (and I would strictly prefer not to add one more tool).

Reply:
FYI I need to this VsCode setting for it to work on save consistently in Puppeteer.
```
"editor.codeActionsOnSave": {
  "source.fixAll.eslint": true
},
 ```

Reply:
> Please make sure this is done by some automatic tool (and I would strictly prefer not to add one more tool).

This is not a new tool: It was already preexisting, AND

Yes, it's done automatically!

Unlike the other one, this one doesn't care about sorting alphabetically, so it doesn't get in the way.